static/js/vue-cdr-access/src/components/full_record/fileDownload.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+    <div v-if="showNonImageDownload" class="actionlink download">
+        <a class="download button action" :href="downloadLink"><i class="fa fa-download"></i> {{ $t('full_record.download') }}</a>
+    </div>
+    <div v-else-if="showImageDownload"
+         class="dropdown actionlink download" :class="{'is-active': show_options}" id="image-download-options">
+        <div class="dropdown-trigger">
+            <button @click="showOptions()" id="download-images" class="button" aria-haspopup="true" aria-controls="dropdown-menu">
+                Download <i class="fas fa-angle-down" aria-hidden="true"></i>
+            </button>
+        </div>
+        <div class="dropdown-menu" id="dropdown-menu" role="menu">
+            <div class="dropdown-content">
+                <a v-if="validSizeOption(800)" :href="imgDownloadLink('800')" class="dropdown-item">Small JPG (800px)</a>
+                <a v-if="validSizeOption(1600)" :href="imgDownloadLink('1600')" class="dropdown-item">Medium JPG (1600px)</a>
+                <a v-if="validSizeOption(2500)" :href="imgDownloadLink('2500')" class="dropdown-item">Large JPG (2500px)</a>
+                <a :href="imgDownloadLink('full')" class="dropdown-item">Full Size JPG</a>
```

Comment:
This should be moved down into the `viewOriginal` permission check, since we are requiring that permission on the server side.